tests/python/gaia-ui-tests/gaiatest/tests/functional/lockscreen/manifest.ini

```diff
@@ -26,3 +26,5 @@ carrier = true
 [test_lockscreen_notification.py]
 
 [test_lockscreen_wake_with_notification.py]
+
+[test_lockscreen_time_check.py]
```

Comment:
Interesting, I tried this myself. I removed my sim cards, but I couldn't get the automatic setting via wifi. In fact, the whole Automatic setting wasn't visible in the UI.